README.md

```diff
@@ -62,6 +63,38 @@ druid's existing functionality and widgets.
 [![flex.rs example](https://raw.githubusercontent.com/xi-editor/druid/screenshots/images/0.6.0/flex.png)](./druid/examples/flex.rs)
 [![custom_widget.rs example](https://raw.githubusercontent.com/xi-editor/druid/screenshots/images/0.6.0/custom_widget.png)](./druid/examples/custom_widget.rs)
 
+## Goals
+
+Druid's primary goal is to provide a smooth and polished user experience with
+pixel perfect design and no behavioral quirks. In order to achieve this we
+strive for a variety of things:
+
+- Make it easy to build and package on all supported platforms.
+- Provide abstractions to avoid platform specific quirks.
+- Encourage good application behavior across platforms.
+- Automate platform resolution independence.
```

Comment:
```suggestion
- Simple and robust handling for display resolution and scaling.
```

(I'm less sure about this wording)

Reply:
All the other items start with verbs, so this change would kind of ruin that vibe. :thinking:

Reply:
Maybe like this:

Handle display resolution and scaling reliably with little effort.